### PR TITLE
add overlord printer

### PR DESCRIPTION
**Pull request by leyap**
sorry for my carelessness before, I have pushed a new branch to add our company's printer. As for the problem on the origin, I resolved it. So this time may be ok.

### CODE REVIEW

**Comment by ghost** [comment]
The Geometry file spark-print-data/data/DreamMaker_OverLord.zip is corrupt somehow print Studio will crash trying to read it. It needs to have a .mtl file as well as the obj. In fact the obj fille. There is no need for the .stp and .igs files

Also you need to increment version "print_definition_revision" in 
https://github.com/spark3dp/print-manager/blob/sandbox/spark-print-data/version.json

You can also delete the database manually when in development.
On Windows: \AppData\Local\Autodesk\Print Studio\printDB
On MacOS /Users//Library/Application Support/com.autodesk.spark.printstudio/printDB

This will make Print Studio read the data you have added

See specific comments on other files. But all tests pass and it does now work!

File: spark-print-mgr/printers/drivers/overlord.js [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+    validateReply : validateReply,
+    expandCode : expandCode,
+    setupExecutor : setupExecutor,
+    updateInterval : 1000
+};
+
+/**
+ * Our module export is a creation function that returns an Ultimaker configured SerialPrinter
+ */
+var createUltimaker = function (data) {
```
Comment (ghost) [inline spark-print-mgr/printers/drivers/overlord.js]: change to createOverlord

File: spark-print-mgr/printers/drivers/overlord.js [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+    return new GcodeClient(data, overlordConfig);
+};
+
+module.exports = createUltimaker;
```
Comment (ghost) [inline spark-print-mgr/printers/drivers/overlord.js]: change to createOverlord

File: spark-print-mgr/printers/drivers/overlord.js [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+    pauseCommands : pauseCommands,
+    getStatusCommands : getStatusCommands,
+    validateReply : validateReply,
+    expandCode : expandCode,
+    setupExecutor : setupExecutor,
+    updateInterval : 1000
+};
+
+/**
+ * Our module export is a creation function that returns an Ultimaker configured SerialPrinter
```
Comment (ghost) [inline spark-print-mgr/printers/drivers/overlord.js]: change Ultimaker to Overlord

File: spark-print-data/version.json [hunk 4]
```diff
@@ -1,4 +1,4 @@
 {
-    "print_definition_schema" : 4,
-    "print_definition_revision" : 8
+    "print_definition_schema" : 5,
```
Comment (ghost) [inline spark-print-data/version.json:2]: this should stay at 4

File: spark-print-data/version.json [hunk 5]
```diff
@@ -1,4 +1,4 @@
...
+    "print_definition_revision" : 1
```
Comment (ghost) [inline spark-print-data/version.json:3]: this should have been integrated to 9

**Comment by ghost** [comment]
Since this has been a long standing pull request I will accept it and fix the minor version problem after the merge.

Thank you for your patience.

**Comment by leyap** [comment]
:+1:

**Pull request merged**